sdk/anomalydetector/ai-anomaly-detector/package.json

```diff
@@ -15,7 +15,7 @@
     "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
     "build:samples": "dev-tool samples prep && cd dist-samples && tsc -p .",
     "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
-    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
+    "build": "npm run lint && tsc -p . && rollup -c 2>&1 && api-extractor run --local",
```

Comment:
There is an analyze step in the CI that will execute the `npm run lint`. We have for the moment decided to rely on that rather than include linting in the `build` script as we have seen this has the potential to increase build times.

The down side is that you wont see the linting error by just running build locally, but the PR CI will definitely fail :)

Reply:
So just by removing the lint report HTML, Analyze should fail when there are lint errors?

Reply:
I just verified it analyze will catch errors in CI :smile:
![image](https://user-images.githubusercontent.com/21109913/91612868-f32b4000-e943-11ea-949f-6027776afe51.png)